models/src/main/java/com/vimeo/networking2/Video.kt

```diff
@@ -190,20 +188,14 @@ data class Video(
      */
     @Json(name = "status")
     @Deprecated("This field is deprecated in favor of upload and transcode.")
-    val status: VideoStatusType? = VideoStatusType.UNKNOWN,
+    val status: VideoStatusType? = null,
 
     /**
      * An array of all tags assigned to this video.
      */
     @Json(name = "tags")
     val tags: List<Tag>? = null,
 
-    /**
-     * The transcode information for a video upload.
-     */
-    @Json(name = "transcode")
-    val transcode: TranscodeType? = TranscodeType.UNKNOWN,
```

Comment:
Why was this property removed?

Reply:
I had removed this property, because it didn't match what the API was returning. The API wraps the transcode status of a transcode object. I created a transcode object with the status.